### ISSUE TITLE
Dyschronometria

### ISSUE THREAD

**Issue by D0ctorNo** [obophenotype/human-phenotype-ontology#10065]
**Preferred term label:**
Dyschronometria

**Synonyms**
Impaired perception of time, Disturbed time perception, Distorted time perception, Lack of time perception, Time distortions, Distorted sense of time

**Definition (free text, please give PubMed ID)**
PMID: 37622046
PMID: 27127597
PMID: 31526769
PMID: 17214571
PMID: 19874739
https://en.wikipedia.org/wiki/Dyschronometria

Dyschronometria is a condition of cerebellar dysfunction in which an individual cannot accurately estimate the amount of time that has passed. It is associated with cerebellar ataxia, when the cerebellum has been damaged and does not function to its fullest ability.

Comment:
Common signs of dyschronometria are often generic to cerebellar ataxia, including a lack of spatial awareness, poor short term memory, and inability to keep track of time. For example, when asked to wait for thirty seconds, or tap every second that has gone by, those affected will be able to perform the task for a short time and then become derailed. This can result from a loss of focus, however, more often than not the individual affected can no longer tell what they are doing and they become disoriented. This often takes form in forgetting basic time keeping unless a timer is set, such as when cooking for example. Dyschronometria does not affect the 24-hour circadian rhythm, which is sustained by a different biological process.

**Parent term (use hpo.jax.org/app)**
HP:0001251 Ataxia
or
HP:0011446 Abnormality of higher mental function
or both?

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**

**Your nano-attribution (ORCID)**

**Comment by pnrobinson** [comment]
@bcoleman96 
thoughts about where this term should go?

**Comment by bcoleman96** [comment]
We have the term [Time agnosia](https://hpo.jax.org/browse/term/HP:5200403). Could we just update that term with the above info?

**Comment by pnrobinson** [comment]
@bcoleman96 I do not think they are synonyms after looking at the literature.

**Comment by pnrobinson** [comment]
@D0ctorNo 

Adding new term: 
- ID: HP:6001044
- label: Dyschronometria
- parents: [Disturbed sensory perception (HP:0010524)](https://hpo.jax.org/app/browse/term/HP:0010524)
- pmid: PMID:37622046|PMID:27127597|PMID:31526769|PMID:17214571|PMID:19874739
- def: Impaired ability to accurately measure or estimate time intervals.
- comment: Common signs of dyschronometria are often generic to cerebellar ataxia, including a lack of spatial awareness, poor short term memory, and inability to keep track of time. For example, when asked to wait for thirty seconds, or tap every second that has gone by, those affected will be able to perform the task for a short time and then become derailed. This can result from a loss of focus, however, more often than not the individual affected can no longer tell what they are doing and they become disoriented. This often takes form in forgetting basic time keeping unless a timer is set, such as when cooking for example. Dyschronometria does not affect the 24-hour circadian rhythm, which is sustained by a different biological process.
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/10065
- synonyms:
